app/pages/lab/workflow-create-form.cjsx

```diff
@@ -26,7 +26,7 @@ WorkflowCreateForm = createReactClass
 
     newWorkflow =
       display_name: @refs.newDisplayName.value
-      primary_language: workflowToClone?.primary_language
+      primary_language: workflowToClone?.primary_language || counterpart.getLocale()
```

Comment:
This is going to re-introduce the bug where workflows are created in the user's current locale eg. German or Arabic, rather than the project's primary language.

Reply:
Confirming that behaviour - i've tested changing a project to a translated language (locale is set away from 'en' default) and then back to the lab - the workflow is then created with the translated locale. 

I think the best solution here is to align the default to the project's `primary_language` vs using counterpart - I'll update accordingly.